src/qae/object-definition.js

```diff
@@ -281,6 +287,11 @@ const objectDefinition = () => {
        */
       refLinesY: [],
     },
+    /**
+     * Array of trend lines
+     * @type {array}
+     */
+    trendLines: [],
```

Comment:
Here I think need to specify what are needed in `trendLines`, check `refLine` for example

Reply:
I did check `refLine`. There is nothing there:
```js
    /**
     * Reference lines settings
     * @type {object}
     */
    refLine: {
      /**
       * Array of x-axis reference line definitions
       * @type {refLine[]}
       */
      refLinesX: [],
      /**
       * Array of y-axis reference line definitions
       * @type {refLine[]}
       */
      refLinesY: [],
    },
```

Reply:
Check `@type {refLine[]}`, if you go down the file, you will see definition.
/**
 * @typedef {object} refLine
 * @property {boolean|ValueExpression} show=true Set to true to display this reference line.
 * @property {string} label Reference line label.
 * @property {paletteColor} paletteColor
 */

/**
 * @typedef {object} refLineExpr
 * @memberof refLine
 * @property {number|ValueExpression} value
 * @property {string|StringExpression} [label]
 */

Reply:
ah, cool!